src/elements/Toggle/Toggle.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Toggle
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Common from '../Common/Common';
+
+const Toggle = ({
+	checked,
+	value,
+	loading,
+	...other
+}) => {
+	const combinedClasses = classNames(
+		'ui-toggle',
+		other.classes,
+		{
+			on: checked,
+			loading
+		}
+	);
+
+	return (
+		<Common
+			{...other}
+			classes={combinedClasses}
+			tag="div"
```

Comment:
This should actually be an `input` with `type` of `checkbox` (this change will probably require a couple style changes)

Reply:
Changed type to checkbox, no warnings or errors.